region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -137,6 +137,55 @@
         "If any Sidehoppers remain after using a Power Bomb, safely kill them from the hole below."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway (Morph Ball Dodge)",
+      "requires": [
+        {"obstaclesNotCleared": ["B"]},
+        "Morph"
+      ],
```

Comment:
So as written, this is in basic. I feel like it should be higher, but I can't think what tech this would be.  We almost need a form of canDodgeEnemies that's in a lower tech.

Reply:
One problem with this in general is that using the runway the player could be expected to tap or stutter. But I think this is lenient enough to not matter. If I'm really careful and a bit lucky, I can get like 6 more tiles